Makefile

```diff
@@ -9,19 +9,24 @@
 # the correct version of tools are installed and environment variables are
 # set appropriately.
 
-SHELL := /bin/bash
+# Prefer simply expanded variables (:=) to avoid confusion caused by recursion.
+# All variables can be overridden in command line by `make TARGET FOO=BAR`.
```

Comment:
nit: lowercase `target` or even `make_target`. Possibly lowercase `bar` too. Conveys naming conventions for different fragments.

Reply:
Done